moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.h

```diff
@@ -84,6 +84,8 @@ class PR2ArmIKSolver : public KDL::ChainIkSolverPos
 
   ~PR2ArmIKSolver() override{};
 
+  void updateInternalDataStructures() override;
```

Comment:
For some reason this function was deleted in (https://github.com/ros-planning/moveit2/pull/60/files#diff-8404d089489122c76cd63a908826c88eL90) even though it overrides a virtual function. Not sure why this didn't complain with eloquent.

Reply:
Removed this since `ros-eloquent-orocos-kdl` is at 3.2.1 which doesn't include this. @clalancette are there plans for making another eloquent release?